src/main/java/core/basesyntax/Fruit.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Fruit {
+    private String name;
+    private Map<LocalDate, Integer> storage;
```

Comment:
It's not very logical that fruit contains storage inside, let's make Class Storage, that will contain fruit